.changeset/friendly-grapes-build.md

```diff
@@ -0,0 +1,12 @@
+---
+"@builder.io/sdk-angular": patch
+"@builder.io/sdk-react-nextjs": patch
+"@builder.io/sdk-qwik": patch
+"@builder.io/sdk-react": patch
+"@builder.io/sdk-react-native": patch
+"@builder.io/sdk-solid": patch
+"@builder.io/sdk-svelte": patch
+"@builder.io/sdk-vue": patch
+---
+
+Fix: check `e.origin` of the message to be a URL first
```

Comment:
@sidmohanty11 For the future: this kind of release note is too technical and does not explain what is actually being fixed. It is not useful for SDK consumers.

A better one would be:

```
Fix: crashes during editing caused by invalid host origin.
```